src/OnlySpans.PolyLeads.Api.Schema/Inputs/Auth/SignInInput.cs

```diff
@@ -4,8 +4,9 @@
 public sealed record SignInInput
 {
     [GraphQLDescription("Ключ для идентификации пользователя, может быть почтой или никнеймом")]
-    public ISignInKey SignInKey { get; init; } = default!;
+    public string Key { get; init; } = default!;
```

Comment:
Если всё таки убираем ISignInKey
```suggestion
    public string Key { get; init; } = string.Empty;
```